rest_framework-stubs/decorators.pyi

```diff
@@ -1,29 +1,13 @@
-from typing import Any, Callable, Optional, Sequence, Type, TypeVar, Union
+from typing import Any, Callable, List, Mapping, Optional, Sequence, Type, Union
 
-from django.http.response import HttpResponseBase
 from rest_framework.authentication import BaseAuthentication
 from rest_framework.parsers import BaseParser
 from rest_framework.permissions import BasePermission
 from rest_framework.renderers import BaseRenderer
 from rest_framework.schemas.inspectors import ViewInspector
 from rest_framework.throttling import BaseThrottle
-
-_VIEW = TypeVar("_VIEW", bound=Callable[..., HttpResponseBase])
-
-def api_view(http_method_names: Optional[Sequence[str]] = ...) -> Callable[[_VIEW], _VIEW]: ...
-def renderer_classes(renderer_classes: Sequence[Type[BaseRenderer]]) -> Callable[[_VIEW], _VIEW]: ...
-def parser_classes(parser_classes: Sequence[Type[BaseParser]]) -> Callable[[_VIEW], _VIEW]: ...
-def authentication_classes(authentication_classes: Sequence[Type[BaseAuthentication]]) -> Callable[[_VIEW], _VIEW]: ...
-def throttle_classes(throttle_classes: Sequence[Type[BaseThrottle]]) -> Callable[[_VIEW], _VIEW]: ...
-def permission_classes(permission_classes: Sequence[Type[BasePermission]]) -> Callable[[_VIEW], _VIEW]: ...
-def schema(view_inspector: Optional[Type[ViewInspector]]) -> Callable[[_VIEW], _VIEW]: ...
```

Comment:
You undid the work that I did in #60, this MR seems reckless.

The result is that argument and return types of views are erased if they use any decorator here.